test/jdk/java/lang/reflect/code/TestPrimitiveTypePatterns.java

```diff
@@ -50,13 +71,30 @@ void testPatternsOfInt(JavaType targetType, int[] values) throws Throwable {
         }
     }
 
+    static Op findOp(Body b, Predicate<Op> p) {
```

Comment:
You can stream the code elements e.g.,
```
var actual = b.elements()
  .filter(ce -> ce instanceof InvokeOp)
  .map(op -> ((InvokeOp) op).invokeDescriptor())   // Use mapMulti instead of filter.map
  .findFirst().orElseThrow();
Assert.assertEquals(actual, expectedConversionMethod);